### PR TITLE
Fix default value for globalIngestionHeapLimitBytes

### PR DESCRIPTION
**Pull request by hevansDev**
Fixes #8255

### Description

Updated the default value of `globalIngestionHeapLimitBytes` to the correct value `1/6th of the JVM heap`

This PR has:

- [x] added documentation for new or modified features or behaviors.
- [x] added comments explaining the "why" and the intent of the code wherever would not be obvious for an unfamiliar reader.

### CODE REVIEW

File: docs/configuration/index.md [hunk 1]
```diff
@@ -1560,7 +1560,7 @@ For most types of tasks, `SegmentWriteOutMediumFactory` can be configured per-ta
...
 |`druid.worker.baseTaskDirSize`|The total amount of bytes that can be used by tasks on any single task dir. This value is treated symmetrically across all directories, that is, if this is 500 GB and there are 3 `baseTaskDirs`, then each of those task directories is assumed to allow for 500 GB to be used and a total of 1.5 TB will potentially be available across all tasks. The actual amount of memory assigned to each task is discussed in [Configuring task storage sizes](../ingestion/tasks.md#configuring-task-storage-sizes)|`Long.MAX_VALUE`|
-|`druid.worker.globalIngestionHeapLimitBytes`|Total amount of heap available for ingestion processing. This is applied by automatically setting the `maxBytesInMemory` property on tasks.|60% of configured JVM heap|
+|`druid.worker.globalIngestionHeapLimitBytes`|Total amount of heap available for ingestion processing. This is applied by automatically setting the `maxBytesInMemory` property on tasks.|Configured JVM heap / 6|
```
Comment (kfaraz) [inline docs/configuration/index.md:1563]:
```suggestion
|`druid.worker.globalIngestionHeapLimitBytes`|Total amount of heap available for ingestion processing. This is applied by automatically setting the `maxBytesInMemory` property on tasks.|Configured max JVM heap size / 6|
```

**Review by kfaraz — APPROVED** [review]

**Comment by hevansDev** [comment]
@kfaraz thanks for the review. There appears to be an issue with the web-check GHA downloading node, would you mind rerunning it to see if that resolves it?

**Comment by kfaraz** [comment]
Thanks for your first contribution to Apache Druid, @hevansDev !

**Comment by hevansDev** [comment]
Thanks @kfaraz! Hopefully the first of many!

**Pull request merged**